### ISSUE TITLE
Simple import format

### ISSUE THREAD

**Issue by adamatan** [IttayEyal/memoization#10]
`from memoize.memoized import Memoized` is long and cumbersome. Let's use something more idiomatic, like `from memorize import Memoize`

**Comment by IttayEyal** [comment]
What should change for that? 
We stay with `@Memoized` for now, so maybe simply `from memoize import Memoized`?

**Comment by IttayEyal** [comment]
Do you know how to simply it? Otherwise plz close issue.

**Comment by adamatan** [comment]
Yes, easily. It's about importing the module from the `__init__.py` file. Will be done when I tackle #3 .
